js/store.js

```diff
@@ -53,6 +53,37 @@ export class StoreClass {
     return undefined
   }
 
+  getMetricDrawState (metricName) {
+    for (const metricBase in this.state.allMetrics) {
+      const metricArray = []
```

Comment:
Please don't use an numbered array here where the developer always has to know which number corresponds to min/avg/max.